docs/src/main/sphinx/installation/cli.rst

```diff
@@ -46,17 +46,59 @@ The recommended TLS implementation is to use a globally trusted certificate. In
 this case, no other options are necessary, since the JVM running the CLI
 recognizes these certificates.
 
+When Trino is configured using custom certificate, invoking CLI requires additional options
+such as ``--keystore-*`` or ``--truststore-*`` to secure TLS connection.
+The simplest way to invoke the CLI is with a wrapper script.
+
 Authentication
 --------------
+Password/LDAP Authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can override your username with the ``--user`` option. It defaults to your
 operating system username. If your Trino server requires password
 authentication, use the ``--password`` option to have the CLI prompt for a
 password. You can set the ``TRINO_PASSWORD`` environment variable with the
 password value to avoid the prompt.
 
+.. code-block:: text
+
+    #!/bin/bash
+    ./trino \
+    --server https://trino-coordinator.example.com:8443 \
+    --catalog <catalog> \
+    --schema <schema> \
+    --user <LDAP user> \
+    --password
+
+Trino CLI options
+-----------------
```

Comment:
@mosabua as you said Trino CLI options should be documented, I found out that this table was available in the LDAP Authentication section, so how about we keep this table as a reference for all CLI options? So I created this new subheading.